README.md

```diff
@@ -97,3 +97,22 @@ The following are common issues encountered when using AWS CloudFormation/CDK wi
    * When you deploy MongoDB Atlas using CloudFormation with your Atlas PAK, CloudFormation will default to use the IP address of the machine that you are making the API call from. 
    * The machine making the API call to the 3rd-party MongoDB Atlas API would be various AWS servers hosting Lambda functions and won't be static. 
    * Review the [AWS IP address ranges](https://docs.aws.amazon.com/general/latest/gr/aws-ip-ranges.html) and contact AWS Support directly who can help confirm the CIDR range to be used in your Atlas PAK IP Whitelist.
+
+### Error: The CFN stack remains in the `CREATE_IN_PROGRESS` state before failing after an hour or so
+The problem is caused by incorrect trust relationships linked to the role used for activating CFN resources or running the CFN stack. To resolve the issue, ensure that your IAM role's trust relationships include resources.cloudformation.amazonaws.com, cloudformation.amazonaws.com, lambda.amazonaws.com. The following YAML code shows an example of the correct trust relationships:
+```yaml
+ AssumeRolePolicyDocument:
+    Version: '2012-10-17'
+    Statement:
+    - Effect: Allow
+      Principal:
+       Service:
+       - lambda.amazonaws.com
+       - resources.cloudformation.amazonaws.com
+       - cloudformation.amazonaws.com
+      Action: sts:AssumeRole
+```
+We suggest using the [execution-role.yaml](examples/execution-role.yaml) file to generate an IAM role that you can use to activate the CFN resources and run your CFN stacks. Alternatively, you can set the trust relationships of your role through the AWS Console. To do so, go to the IAM dashboard, select your role, and click on Trust Relationships.
```

Comment:
avoid `we` when you can. The imperative form is more direct.

another minor edit at the end of the line to confirm to our conventions.
```suggestion
Use the [execution-role.yaml](examples/execution-role.yaml) file to generate an IAM role that you can use to activate the CFN resources and run your CFN stacks. Alternatively, you can set the trust relationships of your role through the AWS Console. To do so, go to the IAM dashboard, select your role, and click **Trust Relationships**.
```